utils/src/main/kotlin/DiskCache.kt

```diff
@@ -35,12 +35,12 @@ class DiskCache(
         /**
```

Comment:
Commit message nit: In Kotlin these are simply called "properties". So maybe say "DiskCache: Use plain constructor parameters where class properties are not needed".

Reply:
Done.